ckanext/canada/controller.py

```diff
@@ -263,6 +265,30 @@ def fgpv_vpgf(self, pkg_id):
             'pkg_id': pkg_id,
         })
 
+    def data_dictionary(self, pd_file):
+        names = pd_file.split('.')
+        if not (len(names)==2 and names[1]=='csv'):
```

Comment:
pep8: there should be spaces around `==`

Python also has str.partition() instead of split to always split once and not raise an error, e.g.

```python
name, dot, ext = pd_file.partition('.')
if ext != 'csv':
   ...
```